content/blog/living-off-the-ipad-as-an-engineer/index.md

```diff
@@ -0,0 +1,110 @@
+---
+{
+    title: 'Living off the iPad as an Engineer',
+    description: "Tips on how to get yourself a proper development environment on the iPad to fully exploit its potential.",
+    published: '2021-02-01T00:00:00.000Z',
+    authors: ['pierremtb'],
+    tags: ['tooling', 'ipad'],
+    attached: [],
+    license: 'cc-by-nc-nd-4'
+}
+---
+Since I transitioned from working all day on my personal MacBook Pro to receiving a work computer for a new engineering position, I decided to go for the only thing financially wise: selling my beloved 16" MacBook Pro to live off the 2018 iPad Pro I had around for iOS app development as **main computer** for my personal life.
+
+While I loved the feeling of knowing that I could open and run anything on the MacBook Pro—aka a conventional laptop—the idea of moving solely to the efficient machine that is the iPad Pro was appealing for various reasons. Yet the question remained: how would I continue the work on side-projects, whether they be software or hardware? We see lots of talking these days on how [LumaFusion](https://freecadweb.org/) is real competition to Adobe Premiere, or that [Affinity Photo](https://affinity.serif.com/en-gb/photo/ipad/) has nothing to envy from desktop Photoshop. While I do spend some time on such creative apps, how am I supposed to maintain [my personal webpage](https://pierrejacquier.com), write code for my Raspberry Pi, or create CAD models for 3D printing?
+
+The answer is mostly through remote access. Fear not, dear reader, we’ll try to rely on tools that are or *feel* native on the iPad Pro, not just cheap Teamviewer-ing. What started as just a 9 to 5 setup challenge, not the other way around, is now much more than that.
```

Comment:
The answer is mostly through remote access. Fear not, dear reader, we’ll try to rely on tools that are native or at least *feel* native on the iPad Pro, not just cheap Teamviewer-ing. What started as just a 9 to 5 setup challenge, not the other way around, is now much more than that.